opus/application/apps/ui/templates/ui/menu.html

```diff
@@ -46,7 +47,11 @@
                                                         {% else %}
                                                             {{ p.body_qualified_label }}
                                                         {% endif %}
-                                                      " data-slug="{{ p.slug }}" class="op-search-param" href="#">
+                                                      "
+                                                      data-dispunit="{% if p.disp_unit %}{{ p.disp_unit }}{% endif %}"
+                                                      data-defaultpunit="{% if p.default_unit %}{{ p.default_unit }}{% endif %}"
```

Comment:
I believe data-defaultpunit is a typo; seems it should likely be data-defaultunit.  Suggestion is to change these like this:
data-dispunit  :     data-dispUnits
data-defaultunit:  data-defaultUnits
data-availunits:     data-availUnits